fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentTest.java

```diff
@@ -1421,7 +1422,7 @@ public void testJoinWithLimitSubQuery5() throws Exception {
 
     @Test
     public void testUnionWithLimitSubQuery() throws Exception {
```

Comment:
Why the test cases need to change？

Reply:
Because limit should be assigned to SetOp, not a separate select in SetOp. select xx limit 10 union all select itself is a syntax error was fixed in the new Parser